core/src/main/java/tc/oc/pgm/commands/AdminCommands.java

```diff
@@ -229,6 +229,25 @@ public void pgm() {
     PGM.get().reloadConfig();
   }
 
+  @Command(
+      aliases = {"reloadpools"},
```

Comment:
I would rather add this functionality to `/pgm`, and no need for any command output. It's just like reloading any other part of the config.

Reply:
Alrighty! I mainly split it up to provide the output, but if it’s not necessary I’ll add this to `/pgm` 👍